tests/test_extensions/test_ext_autodoc_configs.py

```diff
@@ -1421,6 +1425,10 @@ def test_autodoc_type_aliases(app: SphinxTestApp) -> None:
         'io.StringIO': 'my.module.StringIO',
     }
     actual = do_autodoc(app, 'module', 'target.autodoc_type_aliases', options)
+    if sys.version_info >= (3, 14, 0, 'alpha', 7):
+        attr2_typeinfo = ()
```

Comment:
@AA-Turner did you just uncover a bug in `mypy`?  There are two declarations of `attr2_typeinfo`, and only one was adjusted by 059ee9c6269dec3f12eb73c85dd9c6ebb5667427 - yet the [`mypy` lint job](https://github.com/sphinx-doc/sphinx/actions/runs/14978190626/job/42075721516) succeeded.